Introduce JS SDK Docs Section (#1301)
near/docs: docs/sdk/js/contract-structure/collections.md

---
sidebar_position: 2
title: "Collections"
---

# Collections

> When deciding on data structures to use for the data of the application, it is important to minimize the amount of data read and written to storage but also the amount of data serialized and deserialized to minimize the cost of transactions. It is important to understand the tradeoffs of data structures in your smart contract because it can become a bottleneck as the application scales and migrating the state to the new data structures will come at a cost.

Reviewer: ```suggestion
When deciding on data structures to use for the application's data, it is important to minimize the amount of data read and written to storage and the amount of data serialized and deserialized to minimize the cost of transactions. It is important to understand the tradeoffs of data structures in your smart contract because it can become a bottleneck as the application scales, and migrating the state to the new data structures will come at a cost.
```
